src/content/docs/style-guide/formatting/code-examples.mdx

```diff
@@ -173,36 +173,43 @@ To format one or more lines of code, insert three backticks ( ```` ``` ```` ) ab
+  ```js lineHighlight=1,3-5
+  const coolThing = {
+    key: 'HI',
+    tada: 'hello'
+  }
+  // this is an important object
+  ```
 
-That said, `<var>` tags can still be used if you think it's important to highlight something in a code snippet and you're okay losing syntax highlighting. For tips on using var tags, see [Var tags](#var-tags).
+The first line reads: `js lineHighlight=1,3-5`, which tells it to use JavaScript syntax highlighting, and tells it to highlight the first line, and the third through fifth lines. Highlighting lines requires the simultaneous use of syntax highlighting: you must indicate a language if you want to highlight lines. 
 
-Some tips on crafting customer-specific value placeholders: 
+## Placeholders for customer-specific variables [#value-placeholders]
+
+Sometimes we want to use placeholders for variables whose value will depend on the customer or the situation. For example, you might need to use `YOUR_NR_ACCOUNT_ID` to indicate that that's a placeholder for the customer to put in their New Relic account ID.  
+
+Some requirements and tips on crafting customer-specific value placeholders: 
 
-* Use `INSERT` to make it more clear what customers are to do. 
-* Address the reader directly using `YOUR`. For example, use `INSERT_YOUR_NR_ACCOUNT_ID` and not `INSERT_NR_ACCOUNT_ID`.
 * Use all caps and underscores `_` to separate words (also known as [SCREAMING_SNAKE_CASE](https://github.com/rubocop-hq/ruby-style-guide#screaming-snake-case)).
-* Don't combine them with other punctuation to indicate variables (such as wrapping the text in angle brackets or curly braces).
-* If you explicitly say that a customer must replace some values, you can break some of the usual rules. For example, if you preface a code snippet with an instruction that they'll have to replace some of the values, you can use more brief placeholder text, like `YOUR_ACCOUNT_ID` or `YOUR_USER_KEY`. 
-* **Don't** use placeholders for either of these situations: 
-  * Example code or files: if you're showing an example config file or another example that's simply meant to give a rough sense of what something will look like, and there is no direct procedure involved, you don't have to worry about placeholders and can use example values. 
-  * Results of API calls: if you're including the response of a call, you also don't have to use placeholders. 
+* Address the reader directly using `YOUR`. For example, use `YOUR_ACCOUNT_ID` and not `ACCOUNT_ID`.
+* Use `NR` when you think it may not be clear if a component is something of ours or a third party's. For example, when writing about connecting AWS to New Relic, you may want to specific that an account or API key is New Relic's. For example: `YOUR_NR_ACCOUNT_ID`.
```

Comment:
@zuluecho9 I think there's a small typo and that it should be, "...you may want to specify...." Also, I've not see any examples where we're too verbose with placeholders. We're doing our readers a favor by not using abbreviations.